src/deepEqual.js

```diff
@@ -20,7 +20,46 @@
  * @return {boolean}
  */
 function deepEqual(a, b) {
-  // write code here
+  let aContainer = {};
+  let bContainer = {};
+
+  // check the type of variables
+  if ((typeof (a) === 'object')
+    && (a !== null && b !== null)) {
```

Comment:
коли в два рядки if зазвичай форматують ось так:
```suggestion
  if (
    typeof (a) === 'object'
    && (a !== null && b !== null)
  ) {
```